### PR TITLE
Add useBrowserName parameter

### PR DESCRIPTION
**Pull request by darksmile**
I had a problem with karma-junit-reporter - weird browser name in report file name and in class name. I don't need more than one browser and decided to add `useBrowserName`

### CODE REVIEW

**Comment by Dignifiedquire** [comment]
Thanks, can you please make sure travis passes (some linting issues), squash the commits into one and change the commit message to follow our [guidelines](http://karma-runner.github.io/0.13/dev/git-commit-msg.html).

**Comment by darksmile** [comment]
@Dignifiedquire Now working on travis and guidelines. Thanks!

**Comment by darksmile** [comment]
@googlebot, I signed it!

**Comment by brennanfee** [comment]
This would be great.  We get different values on our locals from our CI server so having a consistent path and file name we could control would really help.

**Comment by darksmile** [comment]
@Dignifiedquire, travis fixed, googlebot check passed. Do I have to do something else?

File: index.js [hunk 1]
```diff
@@ -110,7 +118,9 @@ var JUnitReporter = function (baseReporterDecorator, config, logger, helper, for
   this.specSuccess = this.specSkipped = this.specFailure = function (browser, result) {
     var spec = suites[browser.id].ele('testcase', {
       name: result.description, time: ((result.time || 0) / 1000),
-      classname: browser.name.replace(/ /g, '_').replace(/\./g, '_') + '.' + (pkgName ? pkgName + '.' : '') + result.suite[0]
+      classname: (useBrowserName
+        ? browser.name.replace(/ /g, '_').replace(/\./g, '_') + '.'
+        : '') + (pkgName ? pkgName + '.' : '') + result.suite[0]
```
Comment (Dignifiedquire) [inline index.js]: Please pull this out into a function

Comment (darksmile) [inline index.js]: @Dignifiedquire, ready

**Comment by Dignifiedquire** [comment]
Thanks :octocat:

**Comment by darksmile** [comment]
@Dignifiedquire, Thanks!

**Pull request merged**